### PR TITLE
Hapi updated and GET /organizations

### PR DESCRIPTION
**Pull request by AndresSp**
i found a new version available for npm and hapi when i ran my enviroment

### CODE REVIEW

**Comment by fdvj** [comment]
Please rebase your branch from develop like this
```
git checkout develop
git pull upstream develop
git checkout -
git rebase develop
git push -f
```

**Comment by AndresSp** [comment]
but that didnt change anything 
screenshot from 2018-03-26 22-45-11

**Comment by AndresSp** [comment]
My origin and upstream were my forked repository lol, sorry

**Comment by AndresSp** [comment]
How can i separate my last commit, for create a new PR? Well, i will change the title for this PR and include it

File: handlers/organizations.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const Formatters = require('../lib/formatters');
+
+function errorHandler(reply, e) {
+    return this.helpers.errorHandler.apply(this, arguments);
+}
+
+// GET /organizations
+exports.listOrganizations = async function(request, reply) {
+    try {
+        const organizations = await this.models.organization.listOrganizations();
```
Comment (fdvj) [inline handlers/organizations.js:12]: The model always begins with a capital letter. Here it will be `this.models.Organization.listOrganizations()`

**Review by fdvj — CHANGES_REQUESTED** [review]
You are missing the tests. Ill merge this when the tests are included.

You can join commits by doing what is known as a squash. I can do it on merge if you like, or I can show you how to do it from the terminal

**Pull request closed without merging**